README.md

```diff
@@ -4,9 +4,69 @@
 Quay](https://quay.io/repository/rbean/markdown-to-confluence/status "Container Image
 Repository on Quay")](https://quay.io/repository/rbean/markdown-to-confluence)
 
-This is a utility that will convert a directory of markdown files to confluence wiki markup and
-publish them.
+This is a utility that will convert a directory of markdown files to Confluence
+wiki markup and publish them.
 
-## Usage
 
-### podman run -it quay.io/rbean/markdown-to-confluence /bin/bash
+## Authentication
+
+The `CONFLUENCE_USERNAME` and `CONFLUENCE_PASSWORD` environment variables must
+be set to the username and password of an account with appropriate permissions.
+
+
+## Confluence and Markdown
+
+To link to other Confluence documents, use the `confluence:` pseudo-protocol.
+If the spaces are not escaped as `%20`, make sure to wrap the specifier with
+`<>`.
+
+```
+[This text is ignored](<confluence:SPACE:Destination Page Title>)
+```
+
+To link to anchors in the same document, use `#anchor-name` as the link
+destination. To see the name of the anchor for a particular title, inspect the
+dry run output. Anchor elements look like this:
+
+```
+<ac:structured-macro ac:name="anchor"><ac:parameter ac:name="">anchor-name</ac:parameter></ac:structured-macro>
```

Comment:
My adventures in markdown-to-confluence: In [GitHub Flavored Markdown](https://github.github.com/gfm/#html-blocks), the start tag usually needs to be at the beginning of the line. It also needs to be enclosed in `<div>` - this is probably bug in pandoc (perhaps it cannot handle the `:` in tag).

E.g.:

```
| <div><ac:structured-macro ac:name="anchor"><ac:parameter ac:name="">anchor-name</ac:parameter></ac:structured-macro> |
```

results in:

```
| <div>&lt;ac:structured-macro ac:name=&quot;anchor&quot;&gt;&lt;ac:parameter ac:name=&quot;&quot;&gt;anchor-name&lt;/ac:parameter&gt;&lt;/ac:structured-macro&gt;</div> |
```

Reply:
I have also not found a way to pass through Confluence markup directly; it will probably require a change to [confluence.lua](bin/confluence.lua). I added 53043ff to clarify the purpose of the example, which is to show you which text to search for in the `--dry-run` output to ensure you have the right anchor link text (or find it if you couldn't guess it).

Reply:
It's still possible to use Confluence markup in markdown files if it's enclosed in `<div>` and surrounding md is simple enough.

I wrote the test in "tests/test_macro.py" to ensure this doesn't break my usecase otherwise it's hard to figure out when HTML will be escaped and when not.